lib/engine/game/base.rb

```diff
@@ -1022,6 +1022,7 @@ def operating_round(round_num)
           Step::Token,
           Step::Route,
           Step::Dividend,
+          Step::SpecialTrain,
```

Comment:
this should not be added to the base class

Reply:
Fixed